WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -29,12 +30,28 @@ final class SiteAddressViewController: LoginViewController {
                                   keyboardWillHideAction: #selector(handleKeyboardWillHide(_:)))
     }
 
+    /// Localize the "Continue" button
+    ///
     func localizePrimaryButton() {
         let primaryTitle = displayStrings.continueButtonTitle
         submitButton?.setTitle(primaryTitle, for: .normal)
         submitButton?.setTitle(primaryTitle, for: .highlighted)
     }
 
+    /// Registers all of the available TableViewCells
+    ///
+    func registerTableViewCells() {
+        let cells = [
+            InstructionTableViewCell.self
+        ]
+
+        for cell in cells {
+            tableView.register(cell.loadNib(), forCellReuseIdentifier: cell.reuseIdentifier)
+        }
```

Comment:
Just a suggestion - maybe this can less verbose with `forEach`:
`cells.forEach({ tableView.register($0.loadNib(), forCellReuseIdentifier: $0.reuseIdentifier) })`